CRM/Core/BAO/UFMatch.php

```diff
@@ -248,6 +248,9 @@ public static function &synchronizeUFMatch(&$user, $userKey, $uniqId, $uf, $stat
             $primary_email = $user->email;
           }
           $params = ['email' => $primary_email];
+          if (!empty($params['contactID'])) {
```

Comment:
I like the $conflict cleanup but I'm guessing this is a theoretical PR since this will always be empty.

Can I suggest we do the other one for 5.53 since I've run that in multiple scenarios and it works in those, and then we could do any additional changes in master?

Reply:
sounds good to me